pkg/controller/util.go

```diff
@@ -292,8 +295,8 @@ func CreateScratchPersistentVolumeClaim(client kubernetes.Interface, pvc *v1.Per
 // 1. Defined value in CDI config map.
 // 2. If 1 is not available use the 'default' storage class.
 // 3. If 2 is not available use the storage class name of the original pvc that will own the scratch pvc.
-// 4. If none of those are available, fail with an error.
-func GetScratchPvcStorageClass(client kubernetes.Interface, cdiclient clientset.Interface, pvc *v1.PersistentVolumeClaim) (string, error) {
+// 4. If none of those are available, return blank.
+func GetScratchPvcStorageClass(client kubernetes.Interface, cdiclient clientset.Interface, pvc *v1.PersistentVolumeClaim) string {
```

Comment:
Hmmm, I think it may still be a good idea to return an error if we can't get the CDIConfig object.

Reply:
I considered that option, but whomever calls this will ignore that, since a blank is just as good as an actual value. Not being able to find the storage classes is the same as not having a storage class from the perspective of this function, so why make it return something that will just get ignored anyway.

Reply:
I wouldn't say that a blank with no error is just as meaningful as a blank with error.  But maybe we are splitting hairs at this point?

Reply:
I am not that passionate either way, if you really want an error returned as well, I can make that happen. Personally I don't think its that significant either way.

Reply:
NBD either way, but maybe just log an error if we can't get the CDIConfig?

Reply:
Sure I can do that.